README.md

```diff
@@ -42,6 +42,40 @@ A tool to automate front-end development tasks and streamline prototyping.
   * `http://[project].local:8000` if you're running Butler on a Vagrant
   * [http://localhost:8000](http://localhost:8000) if you're developing locally
 
+## Add Butler to a project, with spress
+
+1. Log in to your vagrant environment and navigate to your project. You should always run `npm` commands from within your Vagrant.
```

Comment:
I would add in the commands here: `vagrant up` `vagrant ssh` `cd /var/www/[project].local`. I read this the first time and was like "yeah high fives, good node that it should be run inside vagrant" and then proceeded to _not_ run it inside vagrant. But I could also be especially neglectful.